stdlib/src/hashlib/_ahash.mojo

```diff
@@ -175,6 +205,6 @@ fn hash[
         hash collision statistical properties for common data structures.
     """
 
-    var hasher = AHasher(key)
-    hasher.write(bytes, n)
-    return UInt(int(hasher.finish()))
+    var hasher = AHasher[key]()
+    hasher._update_with_bytes(bytes, n)
+    return UInt(int(hasher^.finish()))
```

Comment:
**Question** Why do we need the `int(...)`?  Also, should we change the return type of this function to be `UInt64` (i.e. match the return type of the `finish()` API)?

Reply:
> Why do we need the `int(...)`?

There is no `__uint__`, so `finish(): UInt64` need to roundtrip to `index` through `int` first.

Reply:
Yeah, this function is temporary it will be replaced with the `_hash_with_hasher` function currently located in `_hasher` module. So I think now it is best to just change the type to be `UInt64`